charts/dns/values.yaml

```diff
@@ -1,26 +1,103 @@
+# Default values for test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1  # Unsatable to set this to more than 1 but I leave it for testing purposes.
+imagePullSecrets: []
+nameOverride: ""
+fullNameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  create: true
+  additionalPermissions: []
+  # - apiGroups: [""]
+  #   resources: ["nodes"]
+  #   verbs: ["list","watch"]
+  # - apiGroups: [""]
+  #   resources: ["pods"]
+  #   verbs: ["get","watch","list"]
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+service:
```

Comment:
As a nit, I'd suggest to rename these services to what they do, as opposed to the protocol they use. For example, `api`, `dns-udp`, `dns-tcp`.

Reply:
Good suggetion. I am doing it.